src/components/countdown/Countdown.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+
+const closedMessage = 'Voting is closed';
+const fifteenMinutes = 15 * 60 * 1000;
+
+// Adapted from https://www.digitalocean.com/community/tutorials/js-building-countdown-timer
+function countdown (d) {
```

Comment:
I think that this section is better served in public/countdown.js because it's pure JavaScript and has nothing to do with React.

I think this is the primary mistake of both React and Angular developers alike: React-ifying perfectly valid, working code, that belongs as vanillajs rather than mangled into a only-works-in-framework-x by superfluous wrappers.

Reply:
I understand that idiomatic React would put these things into a folder `src/lib/countdown.js`, however as this function is only used once, to produce this component, I felt that it was better kept with the component.

Reply:
This is addressed in the latest commit.

I prefer the previous version, with the self-contained Countdown component. I would like to revert that new commit before this is merged, but await advice.